app/api/custom/orders.py

```diff
@@ -167,7 +167,7 @@ def create_order():
     db.session.commit()
     order.populate_and_save()
 
-    return OrderSchema().dumps(order)
+    return OrderSchema().dump(order)
```

Comment:
Why dump instead of dumps?

Reply:
dumps was returning a string -> content/html whereas dump returned dictionary application/json. I saw that in the other places dump had been used.